package.json

```diff
@@ -98,7 +98,6 @@
     "opn-cli": "3.1.0",
     "prettier": "^1.19.1",
     "prop-types": "^15.6.0",
-    "react": "^16.14.0",
```

Comment:
This comes and goes as we run the example site or package the project for NPM. It shouldn't be in the repo normally.